.eslintrc.yml

```diff
@@ -52,6 +52,7 @@ rules:
   import/no-self-import: error
   import/no-useless-path-segments: error
   import/no-webpack-loader-syntax: error
+  import/no-unresolved: off # Doesn't like .js files
```

Comment:
Disabled because it doesn't seem to like resolving '.js' files in a TypeScript codebase.